ci/build.py

```diff
@@ -124,6 +124,9 @@ def tox(tox_args, build_dir):
     tox('-e flake8', build_dir)
 
 if args.run_tests:
+    # Suppress warnings about known memory leaks:
+    os.environ['LSAN_OPTIONS'] = f"suppressions={repo}/ci/lsan-suppressions.txt"
```

Comment:
Is this the best place to set this variable?

Reply:
fine for me.

Reply:
Discussed it with @dnnr too, should be fine.